tf-proj/main.tf

```diff
@@ -17,6 +17,11 @@ resource "buildkite_pipeline" "test" {
   steps = ""
 }
 
+resource "buildkite_organization_settings" "test_settings" {
+  slug = buildkite_pipeline.test.organization_slug
+  allowed_api_ip_addresses = "0.0.0.0/0"
```

Comment:
I want this to be a list instead of a space-delimited string so that users of the provider don't have to do the mapping themselves (other providers output CIDRs as a list so if we accept that it improves usability)